app/helpers/passengers_helper.rb

```diff
@@ -1,13 +1,23 @@
 module PassengersHelper
+
   def expiration_check(passenger)
     if passenger.will_expire_within_warning_period?
       'will_expire_soon'
     elsif passenger.expired_within_grace_period?
       'expired_within_grace_period'
     elsif passenger.expired?
-      'expired'
+      'inactive'
     elsif passenger.temporary? && passenger.expiration.blank?
       'no_note'
     end
   end
+
+  def table_class
```

Comment:
I see that you changed the name of the helper. I was referring to the name of the method.